test/fail_compilation/failCopyCtor.d

```diff
@@ -3,7 +3,7 @@ TEST_OUTPUT:
 ---
 fail_compilation/failCopyCtor.d(10): Error: `struct A` may not define both a rvalue constructor and a copy constructor
 fail_compilation/failCopyCtor.d(12):        rvalue constructor defined here
-fail_compilation/failCopyCtor.d(13):        copy constructor defined here
+fail_compilation/failCopyCtor.d(14):        copy constructor defined here
```

Comment:
Why was this changed?

Reply:
I thought it doesn't matter if the last or first copy constructor is chosen, but you are right, it should be the first.

Reply:
Fixed